website/content/en/scripts/step09-generate-chart.sh

```diff
@@ -0,0 +1,6 @@
+helm template karpenter oci://public.ecr.aws/karpenter/karpenter --version ${KARPENTER_VERSION} --namespace karpenter \
```

Comment:
Why not use `helm install --upgrade --create-namespace`?

Reply:
Okay. Is this right then?
`helm upgrade --install karpenter oci://public.ecr.aws/karpenter/karpenter --version ${KARPENTER_VERSION} --namespace karpenter --create-namespace \`